hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java

```diff
@@ -383,20 +384,19 @@ protected Cluster(
         }
       }
 
+      regionsPerTable = new int[numTables];
```

Comment:
What if we use balance by table, which makes Cluster objects only contains one table ?

Reply:
Thank you for the comments. Sorry I am not sure what the issue is. Could you elaborate? The new changes scale with the number of tables. If there is only one table, numTables=1. only one table is being looked at.